plugins/helpers/bios_selector.rb

```diff
@@ -1,11 +1,10 @@
 def select_bios
-  product = @node_data['lshw']['list']['node']['product']
-  product.downcase!
-  product = product.split(' ')
+  product = @node_data.lshw.list.node.product rescue ''
+  product = product.downcase.split(' ')
 
   # we look for sub templates with increasingly short names
   #   we start with words 1 to n, then 1 to n-1 etc.
-  # this function expects snake case tempaltes stored in a
+  # this function expects snake case templates stored in a
```

Comment:
This LGTM! Thanks, my only gripe is that this comment is still inaccurate (my fault) as it doesn't expect snake case, it expects hyphens. I'll quickly push a change for this then merge it through